diff -Nur -x '*.orig' -x '*.rej' e-smith-formmagick-1.4.0/root/etc/e-smith/locale/en-us/FormMagick/general mezzanine_patched_e-smith-formmagick-1.4.0/root/etc/e-smith/locale/en-us/FormMagick/general
--- e-smith-formmagick-1.4.0/root/etc/e-smith/locale/en-us/FormMagick/general	2008-08-05 19:11:37.000000000 +0200
+++ mezzanine_patched_e-smith-formmagick-1.4.0/root/etc/e-smith/locale/en-us/FormMagick/general	2008-08-05 19:09:55.000000000 +0200
@@ -159,7 +159,7 @@
         </trans>
     </entry>
     <entry>
-        <base>PASSWORD_CHANGE_ERROR</base>
+        <base>ERROR_PASSWORD_CHANGE</base>
         <trans>
             An error occurred while attempting to change your password. Please make sure that the old password entered is correct.
         </trans>
@@ -417,7 +417,7 @@
         </trans>
     </entry>
     <entry>
-        <base>PASSWORD_VERIFY_ERROR</base>
+        <base>PASSWORD_NO_MATCH</base>
         <trans>
             The passwords you entered did not match.
         </trans>
@@ -429,7 +429,7 @@
         </trans>
     </entry>
     <entry>
-        <base>NO_GROUPS</base>
+        <base>ACCOUNT_GROUP_NONE</base>
         <trans>
             There are no user groups in the system.
         </trans>
