diff -Nur -x '*.orig' -x '*.rej' e-smith-formmagick-1.4.0/root/etc/e-smith/locale/en-us/FormMagick/general mezzanine_patched_e-smith-formmagick-1.4.0/root/etc/e-smith/locale/en-us/FormMagick/general
--- e-smith-formmagick-1.4.0/root/etc/e-smith/locale/en-us/FormMagick/general	2008-08-07 19:14:55.000000000 +0200
+++ mezzanine_patched_e-smith-formmagick-1.4.0/root/etc/e-smith/locale/en-us/FormMagick/general	2008-08-07 19:14:14.000000000 +0200
@@ -457,6 +457,10 @@
         </trans>
     </entry>
     <entry>
+      <base>YOUR_ACCOUNT_INVALID</base>
+      <trans>The account name entered is invalid.</trans>
+    </entry>
+    <entry>
         <base>PASSWORD_CHANGE_SUCCESS</base>
         <trans>
             Your password has been successfully changed.
