diff -Nur -x '*.orig' -x '*.rej' e-smith-formmagick-1.4.0/root/etc/e-smith/locale/en-us/FormMagick/general mezzanine_patched_e-smith-formmagick-1.4.0/root/etc/e-smith/locale/en-us/FormMagick/general
--- e-smith-formmagick-1.4.0/root/etc/e-smith/locale/en-us/FormMagick/general	2005-09-30 21:04:39.000000000 +1000
+++ mezzanine_patched_e-smith-formmagick-1.4.0/root/etc/e-smith/locale/en-us/FormMagick/general	2008-02-10 11:44:16.000000000 +1100
@@ -2,6 +2,31 @@
     <!-- vim: ft=xml:
     -->
     <entry>
+        <base>Collaboration</base>
+        <trans>Collaboration</trans>
+    </entry>
+    <entry>
+      <base>Administration</base>
+      <trans>Administration</trans>
+    </entry>
+    <entry>
+        <base>Security</base>
+        <trans>Security</trans>
+    </entry>
+    <entry>
+        <base>Configuration</base>
+        <trans>Configuration</trans>
+    </entry>
+    <entry>
+        <base>Miscellaneous</base>
+        <trans>Miscellaneous</trans>
+    </entry>
+    <entry>
+        <base>Your Settings</base>
+        <trans>Your Settings</trans>
+    </entry>
+
+    <entry>
         <base>OPERATION_STATUS_REPORT</base>
         <trans>Operation status report</trans>
     </entry>
