diff -Nur -x '*.orig' -x '*.rej' e-smith-formmagick-1.3.0/root/etc/e-smith/locale/en-us/FormMagick/general mezzanine_patched_e-smith-formmagick-1.3.0/root/etc/e-smith/locale/en-us/FormMagick/general
--- e-smith-formmagick-1.3.0/root/etc/e-smith/locale/en-us/FormMagick/general	2005-07-17 11:45:00.331458888 -0600
+++ mezzanine_patched_e-smith-formmagick-1.3.0/root/etc/e-smith/locale/en-us/FormMagick/general	2005-07-17 11:40:36.254507296 -0600
@@ -16,4 +16,240 @@
             scroll down and find the specific problem.
         </trans>
     </entry>
+    <entry>
+        <base>FM_NONBLANK</base>
+        <trans>
+            This field must not be left blank
+        </trans>
+    </entry>
+    <entry>
+        <base>FM_INTEGER</base>
+        <trans>
+            This field must contain a positive integer
+        </trans>
+    </entry>
+    <entry>
+        <base>FM_NUMBER</base>
+        <trans>
+            This field must contain a number
+        </trans>
+    </entry>
+    <entry>
+        <base>FM_WORD</base>
+        <trans>
+            This field must look like a single word.
+        </trans>
+    </entry>
+    <entry>
+        <base>FM_DATE</base>
+        <trans>
+            The data entered could not be parsed as a date
+        </trans>
+    </entry>
+    <entry>
+        <base>FM_CREDIT_CARD_NUMBER1</base>
+        <trans>
+            You must enter a credit card number
+        </trans>
+    </entry>
+    <entry>
+        <base>FM_CREDIT_CARD_NUMBER2</base>
+        <trans>
+            Credit card numbers shouldn't have anything but numbers, spaces or dashes
+        </trans>
+    </entry>
+    <entry>
+        <base>FM_CREDIT_CARD_NUMBER3</base>
+        <trans>
+            Must be at least 14 characters in length
+        </trans>
+    </entry>
+    <entry>
+        <base>FM_CREDIT_CARD_NUMBER4</base>
+        <trans>
+            Doesn't appear to be a valid credit card number
+        </trans>
+    </entry>
+    <entry>
+        <base>FM_CREDIT_CARD_EXPIRY1</base>
+        <trans>
+            No expiry date entered.
+        </trans>
+    </entry>
+    <entry>
+        <base>FM_CREDIT_CARD_EXPIRY2</base>
+        <trans>
+            Expiry date must be in the format MM/YY or MM/YYYY
+        </trans>
+    </entry>
+    <entry>
+        <base>FM_CREDIT_CARD_EXPIRY3</base>
+        <trans>
+            This expiry date appears to have already passed
+        </trans>
+    </entry>
+    <entry>
+        <base>FM_CREDIT_CARD_EXPIRY4</base>
+        <trans>
+            This expiry date appears to be too far in the future
+        </trans>
+    </entry>
+    <entry>
+        <base>FM_ISO_COUNTRY_CODE1</base>
+        <trans>
+            You must provide a country code
+        </trans>
+    </entry>
+    <entry>
+        <base>FM_ISO_COUNTRY_CODE2</base>
+        <trans>
+            This field does not contain an ISO country code
+        </trans>
+    </entry>
+    <entry>
+        <base>FM_US_STATE</base>
+        <trans>
+            This doesn't appear to be a valid 2-letter US state abbreviation
+        </trans>
+    </entry>
+    <entry>
+        <base>FM_US_ZIPCODE</base>
+        <trans>
+            US zip codes must contain 5 or 9 numbers
+        </trans>
+    </entry>
+    <entry>
+        <base>FM_MINLENGTH1</base>
+        <trans>
+            Minimum length has been specified meaninglessly as {$minlength}
+        </trans>
+    </entry>
+    <entry>
+        <base>FM_MINLENGTH2</base>
+        <trans>
+            This field must be at least {$minlength} characters
+        </trans>
+    </entry>
+    <entry>
+        <base>FM_MAXLENGTH1</base>
+        <trans>
+            Maximum length has been specified meaninglessly as {$maxlength}
+        </trans>
+    </entry>
+    <entry>
+        <base>FM_MAXLENGTH2</base>
+        <trans>
+            This field must be no more than {$maxlength} characters
+        </trans>
+    </entry>
+    <entry>
+        <base>FM_EXACTLENGTH1</base>
+        <trans>
+            You must specify the length for the field.
+        </trans>
+    </entry>
+    <entry>
+        <base>FM_EXACTLENGTH2</base>
+        <trans>
+            You must specify the exactlength of the field with an integer
+        </trans>
+    </entry>
+    <entry>
+        <base>FM_EXACTLENGTH3</base>
+        <trans>
+            This field must be exactly {$exactlength} characters
+        </trans>
+    </entry>
+    <entry>
+        <base>FM_LENGTHRANGE1</base>
+        <trans>
+            You must specify the maximum and minimum length for the field.
+        </trans>
+    </entry>
+    <entry>
+        <base>FM_LENGTHRANGE2</base>
+        <trans>
+            You must specify the maximum and minimum lengths of the field with an integer
+        </trans>
+    </entry>
+    <entry>
+        <base>FM_LENGTHRANGE3</base>
+        <trans>
+            This field must be between {$minlength} and {$maxlength} characters
+        </trans>
+    </entry>
+    <entry>
+        <base>FM_URL</base>
+        <trans>
+            This field must contain a URL starting with http:// or ftp://
+        </trans>
+    </entry>
+    <entry>
+        <base>FM_EMAIL_SIMPLE1</base>
+        <trans>
+            You must enter an email address.
+        </trans>
+    </entry>
+    <entry>
+        <base>FM_EMAIL_SIMPLE2</base>
+        <trans>
+            This field doesn't look like an RFC822-compliant email address
+        </trans>
+    </entry>
+    <entry>
+        <base>FM_DOMAIN_NAME</base>
+        <trans>
+            This field doesn't look like a valid Internet domain name or hostname.
+        </trans>
+    </entry>
+    <entry>
+        <base>FM_IP_NUMBER1</base>
+        <trans>
+            This field must contain a valid IP number and can not be left blank.
+        </trans>
+    </entry>
+    <entry>
+        <base>FM_IP_NUMBER2</base>
+        <trans>
+            Not enough octets (expected X.X.X.X)
+        </trans>
+    </entry>
+    <entry>
+        <base>FM_IP_NUMBER3</base>
+        <trans>
+            {$octet} is more than 255
+        </trans>
+    </entry>
+    <entry>
+        <base>FM_USERNAME</base>
+        <trans>
+            This field must look like a valid username (3 to 8 letters and numbers)
+        </trans>
+    </entry>
+    <entry>
+        <base>FM_PASSWORD1</base>
+        <trans>
+            You must provide a password.
+        </trans>
+    </entry>
+    <entry>
+        <base>FM_PASSWORD2</base>
+        <trans>
+            The password you provided was not a good password.  A good password must 
+            contain all of the following: upper case letter, lower case letter, number, 
+            non-alphanumeric character, be at least 7 characters long.
+        </trans>
+    </entry>
+    <entry>
+        <base>FM_MAC_ADDRESS1</base>
+        <trans>
+            You must provide a MAC address.
+        </trans>
+    </entry>
+    <entry>
+        <base>FM_MAC_ADDRESS2</base>
+        <trans>
+            The MAC address you provided was not valid.
+        </trans>
+    </entry>
 </lexicon>
