diff -Nur -x '*.orig' -x '*.rej' e-smith-formmagick-1.4.0/root/etc/e-smith/locale/en-us/FormMagick/general mezzanine_patched_e-smith-formmagick-1.4.0/root/etc/e-smith/locale/en-us/FormMagick/general
--- e-smith-formmagick-1.4.0/root/etc/e-smith/locale/en-us/FormMagick/general	2008-08-07 19:10:06.000000000 +0200
+++ mezzanine_patched_e-smith-formmagick-1.4.0/root/etc/e-smith/locale/en-us/FormMagick/general	2008-08-07 19:08:20.000000000 +0200
@@ -189,6 +189,10 @@
         </trans>
     </entry>
     <entry>
+        <base>RESET_PASSWORD_TITLE</base>
+        <trans>Reset user password</trans>
+    </entry>
+    <entry>
         <base>ACCOUNT_PASSWORD_CHANGE</base>
         <trans>
             Change account password
