diff -Nur -x '*.orig' -x '*.rej' e-smith-formmagick-1.4.0/root/etc/e-smith/locale/en-us/FormMagick/general mezzanine_patched_e-smith-formmagick-1.4.0/root/etc/e-smith/locale/en-us/FormMagick/general
--- e-smith-formmagick-1.4.0/root/etc/e-smith/locale/en-us/FormMagick/general	2005-09-30 21:04:39.000000000 +1000
+++ mezzanine_patched_e-smith-formmagick-1.4.0/root/etc/e-smith/locale/en-us/FormMagick/general	2006-04-05 14:08:11.372353283 +1000
@@ -240,6 +240,14 @@
             non-alphanumeric character, be at least 7 characters long.
         </trans>
     </entry>
+
+    <entry>
+        <base>FM_PASSWORD_TOO_LONG</base>
+        <trans>
+	Passwords are limited to a maximum of 14 characters.
+	</trans>
+    </entry>
+
     <entry>
         <base>FM_MAC_ADDRESS1</base>
         <trans>
diff -Nur -x '*.orig' -x '*.rej' e-smith-formmagick-1.4.0/root/usr/lib/perl5/site_perl/esmith/FormMagick.pm mezzanine_patched_e-smith-formmagick-1.4.0/root/usr/lib/perl5/site_perl/esmith/FormMagick.pm
--- e-smith-formmagick-1.4.0/root/usr/lib/perl5/site_perl/esmith/FormMagick.pm	2006-03-16 08:59:29.000000000 +1100
+++ mezzanine_patched_e-smith-formmagick-1.4.0/root/usr/lib/perl5/site_perl/esmith/FormMagick.pm	2006-04-05 14:07:00.615558755 +1000
@@ -794,6 +794,8 @@
 
     my $reason;
 
+    return "FM_PASSWORD_TOO_LONG" if (length $pass > 14);
+
     return "OK" if ($strength eq "none");
 
     $reason = $fm->call_fm_validation('password', $pass, undef);
