diff -Nur -x '*.orig' -x '*.rej' e-smith-formmagick-1.3.2/root/usr/lib/perl5/site_perl/esmith/FormMagick.pm mezzanine_patched_e-smith-formmagick-1.3.2/root/usr/lib/perl5/site_perl/esmith/FormMagick.pm
--- e-smith-formmagick-1.3.2/root/usr/lib/perl5/site_perl/esmith/FormMagick.pm	2005-09-30 21:04:39.000000000 +1000
+++ mezzanine_patched_e-smith-formmagick-1.3.2/root/usr/lib/perl5/site_perl/esmith/FormMagick.pm	2005-12-15 16:43:23.272390526 +1100
@@ -793,30 +793,22 @@
     use Crypt::Cracklib;
 
     my $reason;
-    if ($strength eq 'strong')
-    {
-        $reason = fascist_check($pass, '/usr/lib/cracklib_dict');
-        unless ($reason eq "ok")
-        {
-            $reason = $fm->localise("Bad Password Choice") . ": "
-                . $fm->localise("The password you have chosen is not a good choice, because")
-                . " " . $fm->localise($reason) . ".";
-        }
-    }
-    elsif ($strength eq "none")
-    {
-        $reason = 'OK';
-    }
-    else # assume normal strength checking
-    {
-        $reason = $fm->call_fm_validation('password', $pass, undef);
-    }
+
+    return "OK" if ($strength eq "none");
+
+    $reason = $fm->call_fm_validation('password', $pass, undef);
+
+    return $reason unless ($reason eq "OK");
+    return "OK" unless ($strength eq "strong");
+
+    $reason = fascist_check($pass, '/usr/lib/cracklib_dict');
     $reason ||= "Software error: password check failed";
-    if ($reason eq 'ok' || $reason eq 'OK') {
-        return('OK');
-    } else {
-        return $reason;
-    }
+
+    return "OK" if ($reason eq "ok");
+
+    return $fm->localise("Bad Password Choice") . ": "
+	. $fm->localise("The password you have chosen is not a good choice, because")
+	. " " . $fm->localise($reason) . ".";
 }
 
 =pod
