diff -Nur -x '*.orig' -x '*.rej' e-smith-formmagick-1.4.0/root/etc/e-smith/locale/en-us/FormMagick/general mezzanine_patched_e-smith-formmagick-1.4.0/root/etc/e-smith/locale/en-us/FormMagick/general
--- e-smith-formmagick-1.4.0/root/etc/e-smith/locale/en-us/FormMagick/general	2008-02-10 11:56:30.000000000 +1100
+++ mezzanine_patched_e-smith-formmagick-1.4.0/root/etc/e-smith/locale/en-us/FormMagick/general	2008-02-10 11:55:02.000000000 +1100
@@ -25,7 +25,78 @@
         <base>Your Settings</base>
         <trans>Your Settings</trans>
     </entry>
-
+    <entry>
+        <base>SAVE</base>
+        <trans>Save</trans>
+    </entry>
+    <entry>
+        <base>CANCEL</base>
+        <trans>Cancel</trans>
+    </entry>
+    <entry>
+        <base>ENABLED</base>
+        <trans>Enabled</trans>
+    </entry>
+    <entry>
+        <base>DISABLED</base>
+        <trans>Disabled</trans>
+    </entry>
+    <entry>
+        <base>NO</base>
+        <trans>No</trans>
+    </entry>
+    <entry>
+        <base>YES</base>
+        <trans>Yes</trans>
+    </entry>
+    <entry>
+        <base>ADD</base>
+        <trans>Add</trans>
+    </entry>
+    <entry>
+        <base>CREATE</base>
+        <trans>Create</trans>
+    </entry>
+    <entry>
+        <base>MODIFY</base>
+        <trans>Modify</trans>
+    </entry>
+    <entry>
+        <base>REMOVE</base>
+        <trans>Remove</trans>
+    </entry>
+    <entry>
+        <base>COMMENT</base>
+        <trans>Comment</trans>
+    </entry>
+    <entry>
+        <base>NEXT</base>
+        <trans>Next</trans>
+    </entry>
+    <entry>
+        <base>SELF</base>
+        <trans>Self</trans>
+    </entry>
+    <entry>
+        <base>REMOTE</base>
+        <trans>Remote</trans>
+    </entry>
+    <entry>
+        <base>LOCAL</base>
+        <trans>Local</trans>
+    </entry>
+    <entry>
+        <base>ACTION</base>
+        <trans>Action</trans>
+    </entry>
+    <entry>
+        <base>NETWORK</base>
+        <trans>Network</trans>
+    </entry>
+    <entry>
+        <base>ROUTER</base>
+        <trans>Router</trans>
+    </entry>
     <entry>
         <base>OPERATION_STATUS_REPORT</base>
         <trans>Operation status report</trans>
