diff -Nur -x '*.orig' -x '*.rej' e-smith-formmagick-1.4.0/root/etc/e-smith/locale/en-us/FormMagick/general mezzanine_patched_e-smith-formmagick-1.4.0/root/etc/e-smith/locale/en-us/FormMagick/general
--- e-smith-formmagick-1.4.0/root/etc/e-smith/locale/en-us/FormMagick/general	2008-08-07 20:31:27.000000000 +0200
+++ mezzanine_patched_e-smith-formmagick-1.4.0/root/etc/e-smith/locale/en-us/FormMagick/general	2008-08-07 20:29:14.000000000 +0200
@@ -421,7 +421,7 @@
         </trans>
     </entry>
     <entry>
-        <base>PASSWORD_NO_MATCH</base>
+        <base>PASSWORD_VERIFY_ERROR</base>
         <trans>
             The passwords you entered did not match.
         </trans>
